File: jja6312/2월 2일/음양더하기.java
class Solution {
    public int solution(int[] absolutes, boolean[] signs) {
        int sum = 0;
        for (int i = 0; i < signs.length; i++) {
            if (signs[i]) {
                sum += absolutes[i];
            } else {
                sum -= absolutes[i];
            }
        }

        int answer = sum;
        return answer;
    }
}

File: zuzubibi/2월/2월 2일/음양 더하기.py
def solution(absolutes, signs):
    sum =0 
    for i,j in zip(absolutes, signs):
        if j =='true':
            sum += i
        else:
            sum -= i
    return sum

def mySolution(absolutes, signs):
    return sum( absolute if sign else -absolute for absolute, sign in zip(absolutes, signs))
print(mySolution([4,7,12], ['true','false','true'])) # 9
print(mySolution([1,2,3],['false','false','true'])) # 0